kubernetes-csi-external-provisioner.yaml

```diff
@@ -43,11 +43,12 @@ test:
   pipeline:
     - uses: test/kwok/cluster
     - runs: |
-        csi-provisioner --version
+        csi-provisioner --version | grep "v${{package.version}}"
         mkdir -p /csi
         hostpathplugin --v=5 --endpoint="unix:///csi/csi.sock" --nodeid="node-000000" & sleep 5
         csi-provisioner --v=5 --csi-address "/csi/csi.sock" --kubeconfig ~/.kube/config --http-endpoint ":8080" & sleep 10
-        curl -Lk localhost:8080/metrics
+        code=$(curl -sf -o /dev/null -w '%{http_code}' localhost:8080/metrics)
+        test "$code" -eq 200
```

Comment:
Can we make this verbose? If this fails, unless someone has debug enabled, it won't be clear why

Reply:
updated